spec/spec_helper_acceptance.rb

```diff
@@ -0,0 +1,71 @@
+require 'beaker-rspec/spec_helper'
+require 'beaker-rspec/helpers/serverspec'
+require 'serverspec'
+require 'winrm'
+
+GEOTRUST_GLOBAL_CA = <<-EOM
```

Comment:
This block can be replaced with the cert helper when https://github.com/puppetlabs/beaker/pull/692 is merged :+1: